### ISSUE TITLE
Feature request: Music changes and crossfade

### ISSUE THREAD

**Issue by YukiXXL** [SamClassicPatch/Mod#4]
Music never changes from heavy to medium. Only to peace once all enemies are killed. Can it be "patched" to turn from heavy to medium?

As for crossfade, looping the peace/medium/heavy tracks doesn't sound well. Custom crossfade would be a nice addon or if the current crossfade could be like when changing music.

**Comment by DreamyCecil** [comment]
I don't think this constant change of music is good. And it's rare to see it be used on custom maps, whereas classic campaigns are made in such a way that when the heavy music is activated, it's because an actual heavy battle has begun. Once the battle ends, it resets to normal gameplay. No need to manually activate constant heavy music in this case.

As for the crossfade, I'm currently not planning on enhancing any entities for the editor, only for the game.

**Comment by YukiXXL** [comment]
The constant music change is presented in Next Encounter and suits perfectly so that's why I suggested it but ok if it wouldn't fit the classics.
